q2_types/feature_data/__init__.py

```diff
@@ -47,6 +53,12 @@
     'RNAFASTAFormat', 'AlignedRNAFASTAFormat', 'RNASequencesDirectoryFormat',
     'AlignedRNASequencesDirectoryFormat', 'RNASequence', 'AlignedRNASequence',
     'PairedRNAIterator', 'PairedRNASequencesDirectoryFormat',
-    'PairedEndRNASequence', 'BLAST6Format', 'BLAST6DirectoryFormat', 'BLAST6']
+    'PairedEndRNASequence', 'BLAST6Format', 'BLAST6DirectoryFormat', 'BLAST6',
+    'MixedCaseDNAFASTAFormat', 'MixedCaseDNASequencesDirectoryFormat'
```

Comment:
you missed the comma at the end of this line — this is causing the misleading "imported but not used" error.